stsci/image/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .version import __version__
-from ._image import *
-from .combine import *
+from .version import __version__  # noqa: F401
+from ._image import translate  # noqa: F401, F403
```

Comment:
Is this API change, from `import *` to importing just `translate`?

Reply:
I don't think you have to revert just so long you are aware and okay with the potential behavior change.

I see that `_image.py` has no `__all__` defined, so it would have imported both `_translate` and `translate`.